spec/dummy/app/models/card.rb

```diff
@@ -0,0 +1,35 @@
+require 'time'
+class Card < ActiveRecord::Base
+  belongs_to :block
+  belongs_to :user
+  before_validation :set_review_date_as_now, on: :create
+  validates :user_id, presence: true
+  validates :original_text,
+            :translated_text,
+            :review_date,
+            presence: { message: "Необходимо заполнить поле." }
+  validates :user_id, presence: { message: "Ошибка ассоциации." }
+  validates :block_id,
+            presence: { message: "Выберите колоду из выпадающего списка." }
+  validate :texts_are_not_equal
+
+  scope :pending,
+        -> { where("review_date <= ?", Time.now).order("RANDOM()") }
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.